pkg/kv/kvserver/rangefeed/budget.go

```diff
@@ -49,6 +50,18 @@ var maxFeedFraction = envutil.EnvOrDefaultFloat64("COCKROACH_RANGEFEED_TOTAL_MEM
 var systemRangeFeedBudget = envutil.EnvOrDefaultInt64("COCKROACH_RANGEFEED_SYSTEM_BUDGET",
 	2*64*1024*1024 /* 128MB */)
 
+// RangefeedBudgetsEnabled is a cluster setting that enables rangefeed memory
+// budgets. This is meant to be an escape hatch to disable budgets if they cause
+// feeds to fail unexpectedly while nodes have plenty of memory or if bugs in
+// budgeting are discovered and mitigation is required.
+// With budgets disabled, rangefeeds will behave as they did prior to 22.1.
+var RangefeedBudgetsEnabled = settings.RegisterBoolSetting(
+	settings.SystemOnly,
+	"kv.rangefeed.memory_budgets.enabled",
+	"if set, rangefeed memory budgets are enabled",
```

Comment:
This should mention that it only takes effect after _all_ rangefeeds on a replica have been stopped. Restarting a single rangefeed won't help if there are other rangefeeds on the replica.